book.rb

```diff
@@ -0,0 +1,11 @@
+# book for rental
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] In the same vain, there should be an add_rental method here too for the person, it can look as follow:
```
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```